stable/sumologic-fluentd/Chart.yaml

```diff
@@ -1,5 +1,6 @@
 name: sumologic-fluentd
-version: 0.3.1
+version: 0.4.0
+appVersion: 1.8
```

Comment:
This should be 1.9 as well, then.

Reply:
@timstoop thanks for catching! https://github.com/kubernetes/charts/pull/4539/commits/71cb5641e228548f7f3412ef077bd986a79dbc7d